python/paddle/fluid/framework.py

```diff
@@ -555,7 +555,10 @@ def generated_op_attr_names():
         return {
             core.op_proto_and_checker_maker.kOpRoleAttrName(),
             core.op_proto_and_checker_maker.kOpRoleVarAttrName(),
-            core.op_proto_and_checker_maker.kOpNameScopeAttrName()
+            core.op_proto_and_checker_maker.kOpNameScopeAttrName(),
+            core.op_proto_and_checker_maker.kOpCreationCallstackAttrName(),
+            core.op_proto_and_checker_maker.kOpUseMKLDNNAttrName(),
```

Comment:
why put this here? not every op has mkldnn attr?

Reply:
It is used to skip some attrs when generating doc string for op, and this function `generated_op_attr_names` is only used here.

https://github.com/PaddlePaddle/Paddle/blob/fc87ef741b7e6dc38a3100d4c8d6b9170863efac/python/paddle/fluid/layers/layer_function_generator.py#L91